.github/scripts/arbiter_sortition.py

```
@@ -0,0 +1,175 @@
+    "copilot-pull-request-reviewer[bot]",
+    "coderabbitai[bot]",
+    "qodo-ai[bot]",
+}
+
+HUMAN_REVIEWERS = {"loganfinney27"}
+ALL_REVIEWERS = POTENTIAL_ARBITERS | HUMAN_REVIEWERS
+ARBITER_LABEL_PREFIX = "arbiter/"
+
+
+def _run(cmd, check=True):
+    result = subprocess.run(cmd, capture_output=True, text=True)  # lgtm[py/command-line-injection] - cmd is always a fixed-shape list (never shell=True); pr-number is a validated positive int and label/arbiter values are drawn from ALL_REVIEWERS, a hardcoded set
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)